static/src/javascripts/es6/projects/common/modules/crosswords/clues.js

```diff
@@ -1,9 +1,10 @@
 /* jshint newcap: false */
+import classNames from 'classnames';
+import React from 'react';
+
 import _ from 'common/utils/_';
-import React from 'react/addons';
 
-var classSet = React.addons.classSet,
-    Clue = React.createClass({
+var Clue = React.createClass({
```

Comment:
I'm massively keen to use ES6 classes with React components, but the lack of autobinding seems like it could get annoying, especially with larger scale components, and even worse, has no support for mixins, another large component reusability pattern.

https://facebook.github.io/react/docs/reusable-components.html